notebooks/import_model.ipynb

```diff
@@ -0,0 +1,110 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from transformers import AutoModelForCausalLM, BitsAndBytesConfig, AutoTokenizer\n",
+    "import torch\n",
+    "\n",
+    "print(torch.cuda.is_available())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# model_path = \"codellama/CodeLlama-7b-Instruct-hf\"\n",
+    "# model_path = \"meta-llama/Llama-2-7b-chat-hf\"\n",
+    "model_path = \"./models/meta-llama/Llama-2-7b-chat-hf\"\n",
+    "print(model_path)\n",
+    "target_model_path = \"carlosgjs/Llama-2-7b-chat-hf-4bit\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load the model in 4bit quantization for faster inference on smaller GPUs\n",
+    "conf = {\n",
+    "    \"quantization_config\": BitsAndBytesConfig(\n",
+    "        load_in_4bit=True,\n",
+    "        bnb_4bit_use_double_quant=True,\n",
+    "        bnb_4bit_quant_type=\"nf4\",\n",
+    "        bnb_4bit_compute_dtype=torch.bfloat16,\n",
+    "    ),\n",
+    "    \"device_map\": \"auto\",\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tokenizer = AutoTokenizer.from_pretrained(model_path)\n",
```

Comment:
I would add a comment here for more clarity about what this cell does.

Reply:
done